### PR TITLE
Substreams: Remove cloning

### PR DESCRIPTION
**Pull request by enoldev**

### CODE REVIEW

File: website/pages/en/substreams.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    Ok(BlockMeta {
+        number: blk.number,
+        hash: Hex::encode(&blk.hash),
+        parent_hash: Hex::encode(&header.parent_hash),
+    })
+}
+```
+
+2. **You wrap up your Rust program into a WASM container.** Don't worry if it sounds too complex! You can generate the WASM container just by executing a CLI command.
```
Comment (abourget) [inline website/pages/en/substreams.mdx:25]: into a WASM module using a single command.

File: website/pages/en/substreams.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+3. **The WASM container is sent to a Substreams provider (StreamingFast or Pinax, for example) for execution.** The Substreams provider feeds the WASM container with the blockchain data and the trasformations are applied.
```
Comment (abourget) [inline website/pages/en/substreams.mdx:27]: Substreams endpoint (from SF or Pinax, for example) ?

File: website/pages/en/substreams.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+4. **You select a _sink_, a place where you want to send the transformed data** (a Postgres database or a Subgraph, for example).
```
Comment (abourget) [inline website/pages/en/substreams.mdx:29]: this could link to the `sinks` section of the docs.

File: website/pages/en/substreams.mdx (hunk 2 above)
Comment (abourget) [reply website/pages/en/substreams.mdx:27]: let's remove the references to individual providers, until we have that reachable through the decentralized network and/or the studio

File: nginx.conf [hunk 4]
```diff
@@ -87,12 +87,11 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/arbitrum-faq/$                                $scheme://$http_host/docs/$1/arbitrum/arbitrum-faq/                         permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/migrating-a-subgraph/$               $scheme://$http_host/docs/$1/cookbook/upgrading-a-subgraph/                 permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/quick-start/$                        $scheme://$http_host/docs/$1/quick-start/                                   permanent;
-
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/en/substreams/(.+)$                           https://substreams.streamingfast.io                                         permanent;
```
Comment (benface) [inline nginx.conf:90]:
```suggestion
        rewrite ^/docs/en/substreams/(.+)$                           https://substreams.streamingfast.io                                         permanent;
```

**Review by benface — APPROVED** [review]
Thank you, LGTM

**Pull request merged**